build/stf-run-ci/README.md

```diff
@@ -35,6 +35,8 @@ choose to override:
 | `sg_bridge_repository`                                 | <git_repository>         | https://github.com/infrawatch/sg-bridge               | Which Smart Gateway Bridge git repository to clone                                                                  |
 | `prometheus_webhook_snmp_repository`                   | <git_repository>         | https://github.com/infrawatch/prometheus-webhook-snmp | Which Prometheus webhook snmp git repository to clone                                                               |
 | `loki_operator_repository`                             | <git_repository>         | https://github.com/viaq/loki-operator                 | Which Loki-operator git repository to clone                                                                         |
+| `__service_telemetry_events_endpoint_cert_duration`    | [ParseDuration](https://golang.org/pkg/time/#ParseDuration) | 2160h              | Lifetime of the ElasticSearch endpoint certificate (minimum duration is 1h)   |
+| `__service_telemetry_events_ca_cert_duration`          | [ParseDuration](https://golang.org/pkg/time/#ParseDuration) | 70080h             | Lifetime of the ElasticSearch CA certificate (minimum duration is 1h)         |
```

Comment:
These generally follow the naming pattern in the CRD, so you'll want to update these to `__service_telemetry_events_certificates_endpoint_cert_duration` etc.